src/media/css/site.styl

```diff
@@ -155,6 +155,34 @@ noscript {
     }
 }
 
+.error-page {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    min-height: 85vh;
+
+    > * {
+        max-width: 70%;
+    }
+}
+.error-page-heading {
+    font-weight: 100;
+    margin: 0 0 15px;
+    position: relative;
+
+    &:before {
+        background: url(../img/pretty/alert_triangle.svg) no-repeat 0 0 / 60px auto;
```

Comment:
You brought up a good point for a different reason - 'auto' gets no love from IE or Safari so I'll probably get rid of the background size to begin with - it was there for the previous iteration.